assets/css/app.scss

```diff
@@ -4,3 +4,53 @@
 @import "~bulma/bulma";
 
 @import "explorer";
+
+.alert {
```

Comment:
You could leverage bulma component like this one: https://bulma.io/documentation/elements/notification/

Reply:
I was wondering if the JavaScript DOM manipulation works in the context of Server Side Rendering nature of Phoenix. Do I need to re-render the root layout every time any child elements get removed? If that's the case then why does the initial loading of root layout look like this? 

<img width="1476" alt="Screenshot 2021-09-14 at 01 30 18" src="https://user-images.githubusercontent.com/24897721/133148616-f4462c4f-2520-498d-ae25-76bf3eead860.png">

<img width="599" alt="Screenshot 2021-09-14 at 01 25 58" src="https://user-images.githubusercontent.com/24897721/133148107-db310887-6f30-4473-9f68-38ccd561bf8b.png">

Reply:
Using the default Bootstrap alert css for now.